{{cookiecutter.github_repository_name}}/docs/api/users.md

```diff
@@ -7,57 +7,47 @@ Supports registering, viewing, and updating user accounts.
 
 `POST` `users/`
 
+Parameters:
+
+Name     | Type   | Description
+---------|--------|---
+username | string | The username for the new user.
+password | string | The password for the new user (plaintext).
```

Comment:
Can you please remove `( plaintext ) `here? It should be pretty straight forward since the type is specified as a `string`.